mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -479,6 +483,15 @@ func ResourceComposerEnvironment() *schema.Resource {
 											},
 										},
 									},
+									"web_server_plugins_mode": {
+										Type:         schema.TypeString,
+										Optional:     true,
```

Comment:
Why is this field being added as Optional + Computed instead of having a default value?

Reply:
In case of `workloads_config`, we wanted to avoid having non empty plan after applying config with only part of subfields of workloads_config defined (for example dag_processor subfield is not used in Composer 2). Are there any alternative ways to do this? 
In case `web_server_plugins_mode`, setting this field is not supported in Composer 2. Is it possible to set default conditionaly?

Reply:
As discussed offline, we will make subfields of `workloads_config` Computed, since this field itself is already Computed and not having the subfields Computed sometimes leads to invalid `terraform plan` output.